src/views/InitialScreen/welcome.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="main">
+    <div class="welcome">
+      <img
+        src="/assets/logo.png"
+        alt="Girl in a jacket"
```

Comment:
What kind of alt message is this.

Please follow some standards.

Reply:
Done